airflow_dbt_python/hooks/dbt.py

```diff
@@ -150,12 +149,7 @@ def get_remote(self, scheme: str, conn_id: Optional[str]) -> DbtRemoteHook:
         """
         from .remote import get_remote
 
-        try:
-            return self.remotes[(scheme, conn_id)]
-        except KeyError:
-            remote = get_remote(scheme, conn_id)
-        self.remotes[(scheme, conn_id)] = remote
-        return remote
```

Comment:
From the `functools.cache` docs I interpret that it does roughly the same as this, but also provides thread safety. Given this, I feel indifferent about the change, and I tend to default to accepting changes when that's the case, but I'm curious if there is any value in the change that I'm missing.

Reply:
No any other value